dafni/main_dafni.py

```diff
@@ -0,0 +1,210 @@
+import warnings
+warnings.filterwarnings("ignore", message=".*The 'nopython' keyword.*")
```

Comment:
What warning does this filter out? I tried running the script without this line and didn't see any warnings.